charts/producer-app/templates/service.yaml

```diff
@@ -13,7 +13,10 @@ metadata:
 spec:
   selector:
     app: {{ template "producer-app.name" . }}
-    chart: {{ template "producer-app.chart" . }}
+    release: {{ .Release.Name }}
+    {{- range $key, $value := .Values.labels }}
+    {{ $key }}: {{ $value }}
+    {{- end }}
```

Comment:
Does a selector need to have all pod labels or would `app` and `release` already be enough?

Reply:
Agree with you on that. I just set it to see what feedback comes back.